streams/src/main/java/org/apache/kafka/streams/processor/internals/namedtopology/KafkaStreamsNamedTopologyWrapper.java

```diff
@@ -138,6 +140,29 @@ public void addNamedTopology(final NamedTopology newTopology) {
         topologyMetadata.registerAndBuildNewTopology(newTopology.internalTopologyBuilder());
     }
 
+    /**
+     * Add a new NamedTopology to a running Kafka Streams app. If multiple instances of the application are running,
+     * you should inform all of them by calling {@link #addNamedTopology(NamedTopology)} on each client in order for
+     * it to begin processing the new topology.
```

Comment:
Should we add some description for `topologyExceptionHandler`? ex: `This method will also set the {@code topologyExceptionHandler} for the {@newTopology}`